### ISSUE TITLE
Upgrade to Angula 2?

### ISSUE THREAD

**Issue by billymwlei** [sahat/satellizer#861]
Any timeframe ?

**Comment by josx** [comment]
typescript it is done! now wrapper for angular 1 and angular 2
:+1:

**Comment by sahat** [comment]
Let's keep the discussion in one place. https://github.com/sahat/satellizer/issues/320

TypeScript migration is done, but I haven't tested it out yet. https://github.com/sahat/satellizer/tree/typescript

Once it is merged into `master` branch, writing the Angular 2 wrapper shouldn't be too difficult, I hope.
